### PR TITLE
for #20643

### PR DESCRIPTION
**Pull request by pgaudet**

### CODE REVIEW

File: src/taxon_constraints/only_in_taxon.tsv [hunk 1]
```diff
@@ -812,3 +812,13 @@ GO:0045664	regulation of neuron differentiation	NCBITaxon:33208	Metazoa
...
 GO:0140582	adenylate cyclase-activating G protein-coupled cAMP receptor signaling pathway	NCBITaxon:33083	Dictyostelids
+GO:0031039	macronucleus	NCBITaxon:5878	Ciliophora
+GO:0031040	micronucleus	NCBITaxon:5878	Ciliophora
+GO:0045120	pronucleus	NCBITaxon:33208	Metazoa
+GO:0060418	yolk plasma	NCBITaxon:33208	Metazoa
+GO:0045180	basal cortex	NCBITaxon:33208	Metazoa
+GO:0090652	basolateral cytoplasm	NCBITaxon:33208	Metazoa
+GO:1990794	basolateral part of cell	NCBITaxon:33208	Metazoa
+GO:0016323	basolateral plasma membrane	NCBITaxon:33208	Metazoa
+GO:0044297	cell body	NCBITaxon:33208	Metazoa
```
Comment (balhoff) [inline src/taxon_constraints/only_in_taxon.tsv:823]: Doesn't work for Giardia 'lateral shield'.

Comment (pgaudet) [reply src/taxon_constraints/only_in_taxon.tsv:823]: got it

Comment (pgaudet) [reply src/taxon_constraints/only_in_taxon.tsv:823]: This should fix it https://github.com/geneontology/go-ontology/pull/20760

Comment (pgaudet) [reply src/taxon_constraints/only_in_taxon.tsv:823]: and https://github.com/geneontology/go-ontology/pull/20759

**Pull request merged**